mu-plugins/blocks/global-header-footer/blocks.php

```diff
@@ -183,6 +195,60 @@ function restore_inner_group_container() {
 	}
 }
 
+/**
+ * Render the global header via a REST request.
+ *
+ * @return string
+ */
+function rest_render_global_header( $request ) {
+
+	$markup = render_global_header();
+
+	// Serve the request as HTML
+	add_filter( 'rest_pre_serve_request', function( $served, $result ) {
+		header( 'Content-Type: text/html' );
+
+		echo $result->get_data();
+
+		return $served = true;
+	}, 10, 2 );
+
+	return $markup;
+}
+
+/**
+ * Render the global header via a REST request for the Codex with appropriate tags.
+ *
+ * @return string
+ */
+function rest_render_codex_global_header( $request ) {
+	add_action( 'wp_head', function() {
+		echo '<!-- [codex head meta] -->', "\n";
+	}, 1 );
+
+	add_action( 'wp_head', function() {
+		echo '<!-- [codex head scripts] -->', "\n";
+	}, 100 );
+
+	add_filter( 'body_class', function( $class ) {
+		return [
+			'wporg-responsive',
+			'wporg-codex'
+		];
+	} );
+
+	wp_enqueue_style( 'wp4', 'https://s.w.org/style/wp4.css', array(), 95 );
```

Comment:
I think this is unnecessary after https://github.com/WordPress/wporg-mu-plugins/commit/5cde8ecadd5bfb9a53b1c3cc5e2b7ef6f5c52132

Reply:
That's now handled properly after 9c4143cb0671d583b3cdc55a2a4355a196799730 - https://github.com/WordPress/wporg-mu-plugins/commit/5cde8ecadd5bfb9a53b1c3cc5e2b7ef6f5c52132 wasn't adding the support during rest requests.